tests/integration/llm/prepare.py

```diff
@@ -718,6 +737,16 @@ def build_unmerged_lora_correctness_model(model):
     shutil.copyfile("llm/unmerged_lora.py", "models/test/model.py")
 
 
+def build_ds_smoothquant_model(model):
+    if model not in ds_smoothquant_model_list.keys():
+        raise ValueError(
+            f"{model} is not one of the supporting handler {list(ds_smoothquant_model_list.keys())}"
+        )
+    options = ds_smoothquant_model_list[model]
+    options["engine"] = "DeepSpeed"
```

Comment:
I believe you will also want to specify your handler here options["entryPoint"] = "djl_python.deepspeed" since we aren't using a specific model.py